app/assets/javascripts/users.js

```diff
@@ -0,0 +1,25 @@
+function getRepositories() {
+  var url = $("#repositories").data("repositories")
+  $.getJSON(url, function(data) {
+    $.each(data, function(_idx, repository) {
+      $("#repositories").append("<tr>")
+      $("#repositories").append("<td>" + repository.name + "</td>")
+      $("#repositories").append("<td><a href=\"" + repository.html_url + "\">Go</a></td>")
+      $("#repositories").append("<td>" + repository.stargazers_count + "</td>")
+      $("#repositories").append("<td>" + repository.forks + "</td>")
+      $("#repositories").append("</tr>")
+    })
+  })
+}
+
+function getTemperature() {
+  var url = $("#temperature").data("temp")
+  $.getJSON(url, function(data) {
```

Comment:
It's ok but maybe go with something like: 
```
$.ajax({
  dataType: "json",
  data: data
}).then(
  function.success(data){...},
  function.fail(data){...}  
);
```
To catch what should happend if the request fail, for example